### PR TITLE
LeftNav: use anchor tag for links for accessibility

### PR DESCRIPTION
**Pull request by jonahkagan**
This makes the LeftNav readable for screen readers and tabbable.

### CODE REVIEW

**Comment by CarlNelson** [comment]
Since you're not actually passing an href to these <a> elements, they don't show up as tab accessible.

File: src/LeftNav/NavLink.jsx [hunk 1]
```diff
@@ -7,13 +7,13 @@ export function NavLink(props) {
   const selected = props.selected ? cssClass.SELECTED : null;
 
   return (
-    <li
+    <a
```
Comment (CarlNelson) [inline src/LeftNav/NavLink.jsx]: Since these don't include an href they're not accessible.

Comment (jonahkagan) [inline src/LeftNav/NavLink.jsx]: ah ok i didn't actually understand how to test accessibility. fixed in 51b30fe

**Comment by jonahkagan** [comment]
@CarlNelson and I paired, so this is good to go

**Pull request merged**